medicines/document-manager/stub-api/README.md

```diff
@@ -25,6 +25,50 @@ $ docker run -p 8080:8080 -it --rm stub-document-manager-api
 
 The webserver listens on http://0.0.0.0:8080 by default.
 
+## Deploying to Kubernetes
+
+First, you need to install the `stable/nginx-ingress` and `cert-manager` packages from Helm.
+
+```
+$ export YOUR_IP_ADDRESS=1.2.3.4
+
+$ helm repo add stable https://kubernetes-charts.storage.googleapis.com/
+$ helm repo add jetstack https://charts.jetstack.io
+$ helm repo update
+
+$ helm install api-stub stable/nginx-ingress \
+  --set controller.replicaCount=2 \
+  --set controller.nodeSelector."beta\.kubernetes\.io/os"=linux \
+  --set defaultBackend.nodeSelector."beta\.kubernetes\.io/os"=linux \
+  --set controller.service.loadBalancerIP=$YOUR_IP_ADDRESS \
+  --set controller.service.externalTrafficPolicy=Local
+
+$ kubectl apply --validate=false -f https://raw.githubusercontent.com/jetstack/cert-manager/release-0.12/deploy/manifests/00-crds.yaml
```

Comment:
if we are using this lock version for signing a certificate, would it be possible to move this manifest to our repo and reference?

Reply:
As discussed, due to the ephemeral nature of the stub and the fact that we don't intend to use Helm going forward for Kubernetes dependency management & templating, it's not really a high priority to introduce helm to the stub repo with a full dependency chain & `values.yaml`.